lib/sinatra/base.rb

```diff
@@ -124,7 +124,6 @@ def method_missing(*args, &block)
     end
 
     class MimeTypeEntry
-      attr_reader :params
```

Comment:
Could be a breaking change?

Reply:
no, it wont. MimeTypeEntry is only created and called with accept? here. There is no way that params get called .  https://github.com/sinatra/sinatra/blob/0d7e580133a5bb65b05214be7aa9cf195a4698e9/lib/sinatra/base.rb#L50

Reply:
I was thinking if `Sinatra::Request::MimeTypeEntry` could be considered part of the public interface. I see that `MimeTypeEntry` was just added in v2.1.0 so it is probably safe to make the change.